src/stories/OSCALBackMatter.stories.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import OSCALBackMatter from "../components/OSCALBackMatter";
+import {
+  exampleBackMatter,
+  exampleBackMatterWithCitation,
+  exampleBackMatterWithDescription,
+  exampleBackMatterWithMediaType,
+  exampleBackMatterWithoutTitle,
+} from "../test-data/BackMatterData";
+import { parentUrlTestData } from "../test-data/Urls";
+
+export default {
+  title: "Components/Back Matter",
+  component: OSCALBackMatter,
+};
+
+const Template = (args) => <OSCALBackMatter {...args} />;
+
+export const Default = Template.bind({});
+
+export const BackMatterWithDescription = Template.bind({});
```

Comment:
These variants are all within the Back Matter story so we can remove the `BackMatter` prefix from all of them.